docs/run-api-server/running.mdx

```diff
@@ -5,7 +5,68 @@ sidebar_position: 40
 
 import { CodeExample } from "@site/src/components/CodeExample";
 
-Once your Horizon database and Captive Core configuration is set up properly, you're ready to run Horizon. Run `stellar-horizon` with the [appropriate parameters](./configuring.mdx#parameters) set (or `stellar-horizon-cmd serve` if you [installed via the package manager](./installing.mdx#package-manager), which will automatically import your configuration from `/etc/default/stellar-horizon`), which starts the HTTP server and starts logging to standard out. When run, you should see output similar to:
+Once your Horizon database and Captive Core configuration is set up properly, you're ready to run Horizon.
+
+To run `stellar-horizon` binary direct with the [appropriate parameters](./configuring.mdx#parameters) set (or `stellar-horizon-cmd serve` if you [installed via the package manager](./installing.mdx#package-manager), which will automatically import your configuration from `/etc/default/stellar-horizon`).
+
+To run on Kubernetes using helm chart, ensure you have followed the [pre-requisite](./installing.mdx#helm-chart-installation) of installing the Helm cli tool and adding the Stellar chart repo to helm client.
+
+The horizon process [requires access to a Postgres 12+ database](./configuring.mdx#preparing-the-database), first use the common kubernetes cli tool `kubectl` from your workstation to create a kubernetes secret on the intended namespace of the kubernetes cluster which will hold the horizon database url.
+
+<CodeExample>
+
+```bash
+# copy your horizon DATABASE_URL into a secure file, no line breaks.
+echo -n 'database_url_here' > my_creds.txt
+
+# now generate the kubernetes secret from the file
+kubectl create secret generic \
+-n my-namepsace\
+my-db-secret \
+--from-file=DATABASE_URL=my_creds.txt
+```
+
+</CodeExample>
+
+Now deploy horizon onto the cluster using the helm chart, this example includes usage of `--devel` to enable using the latest development version of published helm chart:
+
+<CodeExample>
+
+```bash
+helm install my-horizon stellar/horizon   \
+--namespace my-namespace  \
+--set ingest.persistence.enabled=true \
+--set web.enabled=true \
+--set web.existingSecret=my-db-secret \
+--set global.image.horizon.tag=2.26.1 \
+--set global.network=testnet \
+--set ingest.existingSecret=my-db-secret \
+--set ingest.horizonConfig.captiveCoreUseDb=true \
+--set ingest.resources.limits.cpu=2 \
+--set ingest.resources.limits.memory=6Gi \
```

Comment:
It might be OK to skip the limits. Many clusters won't have any resource limits and if resources are not specified pods will assume they can burst without any restrictions

Reply:
Ok, I will add some additional statement for users to be aware of and confirm quotas on cluster/namespace and to use these if unsure in any case. As I spent a couple hours debugging failed deployments due to captive core crashes in the pod, and figured out it was due to OOM, and that was after forcing captive core to use db instead of in-memory as well.